src/main/java/org/kiwiproject/registry/eureka/client/EurekaParser.java

```diff
@@ -18,18 +20,24 @@
 
     @SuppressWarnings("unchecked")
     public static List<EurekaInstance> parseEurekaResponse(Map<String, Object> response) {
-        var applications = ((Map<String, Object>) response.get("applications")).get("application");
+        checkArgumentNotNull(response, "Map data of Eureka response must not be null");
```

Comment:
_nitpick:_ Can you change the explanation to "Eureka response map must not be null"?